drlineconfiguration.json

```diff
@@ -1044,35 +1138,53 @@
     "name": "SABIO-RK",
     "displayName": "SABIO-RK",
     "category": "EAP",
-    "uriLink": "http://sabiork.h-its.org/newSearch?q=UniProtKB_AC:%u"
+    "uriLink": "http://sabiork.h-its.org/newSearch?q=UniProtKB_AC:%id"
   },
   {
     "name": "SignaLink",
     "displayName": "SignaLink",
     "category": "EAP",
-    "uriLink": "http://signalink.org/protein/%s"
+    "uriLink": "http://signalink.org/protein/%id"
   },
   {
     "name": "SIGNOR",
     "displayName": "SIGNOR",
     "category": "EAP",
-    "uriLink": "https://signor.uniroma2.it/relation_result.php?id=%u"
+    "uriLink": "https://signor.uniroma2.it/relation_result.php?id=%id"
   },
   {
     "name": "UniPathway",
     "displayName": "UniPathway",
     "category": "EAP",
-    "uriLink": "http://www.unipathway.org?upid=%s&entryac=%u",
+    "uriLink": "http://www.unipathway.org?upid=%id&entryac=%accession",

```

Comment:
These links are dead: http://www.unipathway.org/?upid=UPA00143&entryac=Q80XJ2

Reply:
no link, can we set this as "unknown"